tests/tests/test_contacts/test_mixing.py

```diff
@@ -0,0 +1,23 @@
+import phoebe
+import numpy as np
+
+
+def test_mixing(verbose=False):
+    b = phoebe.default_contact_binary()
+    b.add_dataset('mesh', compute_times=[0])
+    b.set_value('teff@secondary', 3000)  # so we have different temperatures
+    b.set_value('mixing_enabled', True)
+    b.set_value('mixing_method', 'perfect')  # since other mixing methods are arbitrarily parametrized, test this case only
+    b.set_value('columns', '*')  # to populate 'teffs'
+
+    assert b.run_checks().passed
```

Comment:
I'm guessing that, eventually, we'll be able to do better than just test passing checks? It would also be good to find a way to test other mixing methods, but that shouldn't hold up this PR.